BlueprintUICommonControls/Tests/Sources/AttributedLabelTests.swift

```diff
@@ -5,6 +5,63 @@ import XCTest
 
 class AttributedLabelTests: XCTestCase {
 
+    func test_accessibilityTraits() {
+
+        let string = NSAttributedString(string: "Hello, World!")
+
+        let nilTraits = AttributedLabel(attributedText: string) { label in
+            label.accessibilityTraits = nil
+        }
+
+        let noTraits = AttributedLabel(attributedText: string) { label in
+            label.accessibilityTraits = []
+        }
+
+        let header = AttributedLabel(attributedText: string) { label in
+            label.accessibilityTraits = [.header, .staticText]
+        }
+
+        let headerAndStatic = AttributedLabel(attributedText: string) { label in
+            label.accessibilityTraits = [.header, .staticText]
+        }
+
+        let updatesFrequently = AttributedLabel(attributedText: string) { label in
+            label.accessibilityTraits = [.updatesFrequently]
+        }
+
+        let view = BlueprintView(frame: CGRect(x: 0, y: 0, width: 100, height: 100))
+
+        nilTraits.accessBackingView(in: view) { label in
+            let label = label as! UILabel
+
+            XCTAssertEqual(label.accessibilityTraits, [])
```

Comment:
Hmm actually I'm surprised this is empty, vs being `.staticText`.

Reply:
Any reason I _shouldn't_ make this be `.staticText`?

Reply:
Alternatively make `[.staticText]` the default traits on the model instead of `nil`.

Reply:
I'm going to go with the former – include static text if the traits are nil. If the traits are `[]`, that means they were explicitly set to empty.